.github/workflows/integration-test.yml

```diff
@@ -44,12 +41,10 @@ jobs:
         run: npm test
         
       - name: Trigger harvest and verify completion
-        if: ${{ inputs.runType == 'dynamic' || 'all' }}
-        run: DYNAMIC_COORDINATES=true npm run e2e-test-harvest
+        run: DYNAMIC_COORDINATES=${{ matrix.dynamicCoordinates }} npm run e2e-test-harvest
 
       - name: Verify service functions
-        if: ${{ inputs.runType == 'fixed' || 'all' }}
-        run: DYNAMIC_COORDINATES=false npm run e2e-test-service
```

Comment:
Between steps `Trigger harvest and verify completion` and `Verify service functions`, the value of `DYNAMIC_COORDINATES` must be the same. This is what's caused the problem in the previous change, where the values were different (first true, then false). FYI @elrayle @ljones140 .

Reply:
Thanks @RomanIakovlev